Source/AliveLibAO/BaseAnimatedWithPhysicsGameObject.cpp

```diff
@@ -47,9 +55,46 @@ BaseAnimatedWithPhysicsGameObject* BaseAnimatedWithPhysicsGameObject::ctor_417C1
     return this;
 }
 
-void BaseAnimatedWithPhysicsGameObject::Animation_Init_417FD0(int /*frameTableOffset*/, int /*maxW*/, int /*maxH*/, BYTE** /*ppAnimData*/, __int16 /*a6*/)
+void BaseAnimatedWithPhysicsGameObject::Animation_Init_417FD0(int frameTableOffset, int maxW, int maxH, BYTE** ppAnimData, __int16 a6)
 {
-    NOT_IMPLEMENTED();
+    const auto init = field_10_anim.Init_402D20(
```

Comment:
might as well drop that var and put the call directly within in the if ?

Reply:
just put it there because it reads a bit better imo /shrug

Reply:
if you think its better then leave it as-is, its subjective :)